### ISSUE TITLE
MoveToTrash error

### ISSUE THREAD

**Issue by ndcooke** [fman-community/fman#335]
I am unable to delete files or folders under Windows, including files of folders under my documents, or folders created by fman the operation before.

I receive "MoveToTrash" error.

![image](https://user-images.githubusercontent.com/14249205/36741883-e3fc46dc-1bab-11e8-9e22-71a1f32c50c7.png)

**Comment by mherrmann** [comment]
Thanks for the report and sorry for the trouble Norman. I updated fman's Python version in 0.9.1 and there seem to be several problems related to this. I'm in the process of drafting a new fman release that restores the "old" Python version. Hopefully, it will fix the problem. Will let you know once it's out.

**Comment by mherrmann** [comment]
fman 0.9.2 is out. Does the problem still appear when you [update](https://fman.io/docs/automatic-updates)?

**Comment by ndcooke** [comment]
I just updated and I still get the same error.

**Comment by mherrmann** [comment]
Okay, sorry. Then I don't think it's caused by the new version. I'll take a look tomorrow.

**Comment by ndcooke** [comment]
Oh, bummer.  Sorry to have you revert a python upgrade and then it not do anything.

I'm available for troubleshooting it whenever.  I'll test it on my other systems when I get home to see if maybe it's a configuration issue from an old plug-in (I removed all of them to troubleshoot the issue).

I'm not sure what configuration files I'd need to drop from where to do a clean reinstall to test whether it's a lingering condition of something that was changed earlier.
